app/controllers/api/v1/admin_dashboard_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::AdminDashboardController < ApplicationController
+  # GET /api/v1/admin_dashboard
+  def index
+    users_count = User.count
+    kit_requests_count = KitRequest.count
+    total_donations = Donation.sum(:amount)
+    if current_user.role === "admin"
```

Comment:
```suggestion
    if current_user.role == "admin"
```

Ruby's `===` is quite different from JavaScript's. Instead of strict equality, Ruby's is generally used for [case expressions](https://ruby-doc.com/docs/ProgrammingRuby/html/tut_expressions.html#S5), or has a special meaning for certain classes that is IMO better to avoid in favour of more descriptive methods. For example:

* To check whether an integer falls within a range, `(1..10) === 5` is not as clear as `(1..10).include?(5)`
* When checking whether a regular expression matches, `/foo.*bar/ === "foo - bar"` is not as clear as `/foo.*bar/.match?("foo - bar")`.)

I've run into some gotchas related to `===` myself, so I recommend sticking to `==` in Ruby in general unless you are familiar with the implementation of `===` for every class that you use it on, and prefer not to use more descriptive methods that accomplish the same thing.